easybuild/easyconfigs/a/arosics/arosics-1.7.6-foss-2021a.eb

```diff
@@ -0,0 +1,80 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PythonBundle'
+
+name = 'arosics'
+version = '1.7.6'
+
+homepage = 'https://github.com/GFZ/arosics'
+description = """AROSICS is a python package to perform automatic subpixel co-registration
+of two satellite image datasets based on an image matching approach working in the frequency
+domain, combined with a multistage workflow for effective detection of false-positives."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('GEOS', '3.9.1'),
+    ('GDAL', '3.3.0'),
+    ('Pillow', '8.2.0'),
+    ('matplotlib', '3.4.2'),
+    ('scikit-image', '0.18.3'),
+    ('Fiona', '1.8.20'),
+    ('Cartopy', '0.20.0'),
+    ('pyproj', '3.1.0'),
+    ('dill', '0.3.4'),
+    ('networkx', '2.5.1'),
+    ('plotly.py', '5.1.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('fonttools', '4.29.1', {
+        'modulename': 'fontTools',
+        'source_tmpl': SOURCE_ZIP,
+        'checksums': ['2b18a172120e32128a80efee04cff487d5d140fe7d817deb648b2eee023a40e4'],
+    }),
+    ('PyKrige', '1.6.1', {
+        'checksums': ['7432287306d13820ed0f0ec2f52c12fa462e18015abbf8fa627beab989ccaff6'],
+    }),
+    ('pyFFTW', '0.12.0', {
+        'source_tmpl': SOURCE_TAR_GZ,
+        'checksums': ['60988e823ca75808a26fd79d88dbae1de3699e72a293f812aa4534f8a0a58cb0'],
+    }),
+    ('spectral', '0.22.4', {
+        'checksums': ['b208ffd1042e32fd2276a35e098e3df26a5f6ff1310b829e97d222c66645a9af'],
+    }),
+    ('py-tools-ds', '0.20.1', {
+        'source_tmpl': 'py_tools_ds-%(version)s.tar.gz',
+        'checksums': ['c745a3e540c2b75dbffc1783df4307707b0b9f80cc7f30de4f1138983f62d899'],
+    }),
+    ('geopandas', '0.10.2', {
+        'checksums': ['efbf47e70732e25c3727222019c92b39b2e0a66ebe4fe379fbe1aa43a2a871db'],
+    }),
+    ('geojson', '2.5.0', {
+        'checksums': ['6e4bb7ace4226a45d9c8c8b1348b3fc43540658359f93c3f7e03efa9f15f658a'],
+    }),
+    ('geoarray', '0.15.7', {
+        'source_tmpl': SOURCE_TAR_GZ,
+        'checksums': ['65678c49332ff471d1c95a26734ee67086fb02490d437647e9de67571022532f'],
+    }),
+    ('branca', '0.4.2', {
+        'checksums': ['c111453617b17ab2bda60a4cd71787d6f2b59c85cdf71ab160a737606ac66c31'],
+    }),
+    ('folium', '0.12.1.post1', {
+        'checksums': ['e91e57d8298f3ccf4cce3c5e065bea6eb17033e3c5432b8a22214009c266b2ab'],
+    }),
+    ('cmocean', '2.0', {
+        'checksums': ['13eea3c8994d8e303e32a2db0b3e686f6edfb41cb21e7b0e663c2b17eea9b03a'],
+    }),
+    (name, version, {
+        'source_tmpl': SOURCE_TAR_GZ,
+        'checksums': ['03f4f952cca5d9d709da1a3c2130ddb3fbb1d21b83998593569638a4b1c7aaa9'],
+    }),
+]
+
+moduleclass = 'devel'
```

Comment:
any particular reason for choosing `devel`? I'd think `geo` is probably a more suitable fit

Reply:
I don't think I had a particular reason. Probably a leftover from the EC I started with.

Reply:
@jfgrimm Done!